Package.swift

```diff
@@ -35,7 +36,10 @@ let package = Package(
         ),
         .testTarget(
             name: "xcodeinstallTests",
-            dependencies: ["xcodeinstall"],
+            dependencies: [
+                "xcodeinstall",
+                .product(name: "Testing", package: "swift-testing")
```

Comment:
as said above, no need to have a dependency on swift-testing, migrate `Package.swift` to Swift 6 instead